### PR TITLE
Rebound, adding files for VIA compat

### PR DESCRIPTION
**Pull request by Rossman360**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/montsinger/rebound/rev4/config.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT_ID 0x552F
 #define DEVICE_VER 0x0002
 #define MANUFACTURER Montsinger
-#define PRODUCT Rebound
+#define PRODUCT Montsinger Rebound
```
Comment (drashna) [inline keyboards/montsinger/rebound/rev4/config.h:25]:
```suggestion
#define PRODUCT Rebound
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/montsinger/rebound/rev4/config.h [hunk 2]
```diff
@@ -22,7 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 #define DESCRIPTION "A conjoined Let's Split"
```
Comment (fauxpark) [inline keyboards/montsinger/rebound/rev4/config.h:26]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/montsinger/rebound/rev4/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,102 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/montsinger/rebound/rev4/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]
Thanks!

**Comment by drashna** [comment]
Also, it looks like the `ortho_4x12` layout is erroring out on the rev4 stuff.  That's outside the scope of this PR, but is something that should be fixed, eventually.

**Pull request merged**